ssv/committee_runner.go

```diff
@@ -38,11 +39,12 @@ func NewCommitteeRunner(beaconNetwork types.BeaconNetwork,
 			Share:          share,
 			QBFTController: qbftController,
 		},
-		beacon:         beacon,
-		network:        network,
-		signer:         signer,
-		operatorSigner: operatorSigner,
-		valCheck:       valCheck,
+		beacon:          beacon,
+		network:         network,
+		signer:          signer,
+		operatorSigner:  operatorSigner,
+		valCheck:        valCheck,
+		submittedDuties: make(map[types.BeaconRole]map[phase0.ValidatorIndex]map[phase0.Slot]struct{}),
```

Comment:
to simplify the submittedDuties map, i think we can remove the slot, bc its the same for the whole lifecycle, and instead just reset the map when StartDuty is called

Reply:
also we have committeeRunners per slot

Reply:
Done. Thanks!